src/clean-core/assert.hh

```diff
@@ -16,23 +16,25 @@
 // CC_ENABLE_ASSERTIONS enables assertions
 // CC_ENABLE_BOUND_CHECKING enables bound checking
 
+#define RUNTIME_ASSERT(condition) \
+    (CC_UNLIKELY(!(condition)) ? ::cc::detail::assertion_failed({#condition, CC_PRETTY_FUNC, __FILE__, __LINE__}) : void(0)) // force ;
+
 #if !defined(CC_ENABLE_ASSERTIONS)
 #define ASSERT(condition) CC_UNUSED(condition)
 #else
 #define ASSERT(condition) \
     (CC_UNLIKELY(!(condition)) ? ::cc::detail::assertion_failed({#condition, CC_PRETTY_FUNC, __FILE__, __LINE__}) : void(0)) // force ;
 #endif
```

Comment:
hm I'm wondering if I would prefer `CC_ASSERT` not. My younger self had some reason to name it `ASSERT` but I don't remember it anymore.